__tests__/regression/fixtures/indented-content-is-not-code/body.md

```
@@ -0,0 +1,25 @@
+
+    * Wildcards are supported in file names as follows:
+      * **Suppress name lookup** must be enabled.
+
+6. **Use Authentication**
+</Accordion>
+
+Fenced code still works, including when indented inside a list:
+
+1. Install the CLI:
+
+   ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)